robot/lib/Inc/drivers/Battery.hpp

```diff
@@ -1,28 +1,47 @@
 #pragma once
 
 #include "cstdint"
-
+/** @class Battery
+ * Interfaces with analog pin to get voltage-related data
+ *
+ * @note As of May 2020, this class does not currently work, as it does not read analog pin
+ */
 class Battery {
 public:
     Battery();
 
-    void update();
+    void update();             /**< Updates reading from analog pin*/
 
-    float getBattPercentage();
+    float getBattPercentage(); /**< Returns the last battery percentage read */
 
-    uint8_t getRaw();
+    uint8_t getRaw();          /**< Returns battery voltage as a raw 8-bit integer */
 
-    bool isBattCritical();
+    bool isBattCritical();     /**< Stores whether battery is critical (at or below 0%) */
```

Comment:
Is this the same as `@return`?

Reply:
I accidentally treated this as a variable, but it is a function. I'll put returns instead (rather than stores)